src/coreclr/src/inc/corinfo.h

```diff
@@ -407,7 +407,6 @@ enum CorInfoHelpFunc
     CORINFO_HELP_NEW_MDARR,         // multi-dim array helper (with or without lower bounds - dimensions passed in as vararg)
     CORINFO_HELP_NEW_MDARR_NONVARARG,// multi-dim array helper (with or without lower bounds - dimensions passed in as unmanaged array)
     CORINFO_HELP_NEWARR_1_DIRECT,   // helper for any one dimensional array creation
-    CORINFO_HELP_NEWARR_1_R2R_DIRECT, // wrapper for R2R direct call, which extracts method table from ArrayTypeDesc
```

Comment:
Since this updated corinfo.h, this should have also revved the JitEEVersionIdentifier: see https://github.com/dotnet/runtime/blob/master/docs/project/updating-jitinterface.md

Reply:
A PR to update the guid: https://github.com/dotnet/runtime/pull/1330